src/tx/script/script-engine.ts

```
@@ -0,0 +1,443 @@
+
+    this.astack = new DataStack();
+    this.numOps = 0;
+  }
+
+  execute(): void {
+    let scripts = [];
+    let isP2shVal = false;
+
+    switch (this.scriptSource.type) {
+      case 'TxInput':
+        const { input, utxoEntry, isP2sh } = this.scriptSource;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)